ldes-server-port-fetch-rest/src/test/java/be/vlaanderen/informatievlaanderen/ldes/server/rest/treenode/services/TreeNodeConverterImplTest.java

```diff
@@ -63,7 +63,7 @@ void when_TreeNodeHasNoMembersAndIsAView_ModelHasTreeNodeAndLdesStatements() {
 
         Model model = treeNodeConverter.toModel(treeNode);
 
-        assertThat(model.listStatements().toList()).hasSize(24);
+        assertThat(model.listStatements().toList()).hasSize(25);
```

Comment:
can you use `model.size()` instead of converting the model to an iterator and then to a list?